README.md

```diff
@@ -4,11 +4,20 @@
 
 ### Disclaimer: work in progress and not ready for production use. 
 
-kube-bind is a prototype project with the goal to establish a new extension model for Kubernetes clusters:
+You are invited to [contribute](#contributing)!
 
-- APIs should be bindable into a cluster and operated by a service provider
-- these APIs should not require (custom) controllers/operators run locally in the consuming cluster
-- only a single vendor-neutral, OpenSource agent should be required.
+## What is it?
+
+kube-bind is a prototype project that aims to provide better support for service providers and consumers that reside in distinct Kubernetes clusters.
+
+- A service provider defines its API in terms of CRDs and associated permission claims/limitations, and exports it for use from other clusters.
+- Service consumers identify the services they want to consume.
+- The service CRDs get installed in the service consumer clusters, with objects of the defined kinds written and read by the service consumers.
+- The service provider indirectly reads and writes those objects as the interface to the service that it provides.
```

Comment:
```suggestion
- The service provider indirectly reads and writes(based on the permission given) those objects as the interface to the service that it provides.
```

Reply:
Is this even true?  I have a concern in the opposite direction: that word "indirectly" in my proposal is covering a lot of information --- the fact that there are copies of the service API objects in the service provider's cluster and the konnector is copying state back and forth between the two copies of a given service API object.

Reply:
Consumer creates object on the consumer cluster and then konnector creates it in the Provider side. Konnector does status down-sync(provider->consumer) & spec up-sync(consumer).

Reply:
Provider doesn't write anything in the spec on the Consumer side. So write seems bit generic to me.

Reply:
btw, let's merge it. much better info than before